### PR TITLE
Add vendor related functionality

### PR DESCRIPTION
**Pull request by chandankumar4**
Signed-off-by: Chandan Kumar <chandan.kumar@mayadata.io>

**What this PR does / why we need it**:
- Add vendor related key value pair in language file
- If vendor is enabled( vendor: true ) then openebs specific functionality get hidden
and vice versa

**Special notes for your reviewer**:

vendor: false
image

vendor : true
image

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -7,4 +8,10 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'dashboard-ci';
+  public vendor: any = false;
+  constructor(public translateService: TranslateService) {
+    this.translateService.getByFileName('VENDOR', 'default-en').subscribe(value => {
```
Comment (yudaykiran) [inline src/app/app.component.ts]: Can we have buttons in the UI for language selection and set the language based on that?

Comment (chandankumar4) [reply src/app/app.component.ts]: Current there is only one language file! so we can't give button now, later if we increase the language file then we can put the button!!

File: src/assets/languages/default-en.json [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {
+  "VENDOR" : "false",
   "SIDEBAR": {
     "HOME": "Overview",
     "DASHBOARD": "Ci/E2E Dashboard",
     "WORKLOADS": "Workload Dashboard",
-    "ORGANISATION": "MayaData.Inc"
+    "ORGANISATION": "MayaData.Inc",
+    "DASHBOARD-URL": "https://gitlab.com/geckovuser/"
```
Comment (inyee786) [inline src/assets/languages/default-en.json]: this is not need here

Comment (chandankumar4) [reply src/assets/languages/default-en.json]: This link is not used if vendor is false, we can pass this ant link or blank also

**Pull request merged**